chaoscenter/authentication/api/handlers/rest/user_handlers.go

```diff
@@ -57,6 +59,19 @@ func CreateUser(service services.ApplicationService) gin.HandlerFunc {
 			c.JSON(utils.ErrorStatusCodes[utils.ErrInvalidRequest], presenter.CreateErrorResponse(utils.ErrInvalidRequest))
 			return
 		}
+		//username validation
+		err = utils.ValidateStrictUsername(userRequest.Username)
+		if err != nil {
+			c.JSON(utils.ErrorStatusCodes[utils.ErrStrictUsernamePolicyViolation], presenter.CreateErrorResponse(utils.ErrStrictUsernamePolicyViolation))
+			return
+		}
+
+		// password validation
+		err = utils.ValidateStrictPassword(userRequest.Password)
```

Comment:
For create user we can remove the password validation since user will be resetting it anyways.

Reply:
do u mean the user will manually reset it?